### ISSUE TITLE
Nested array validation not working

### ISSUE THREAD

**Issue by Mezatsong** [Mezatsong/laravel-swagger-docs#6]
![image](https://user-images.githubusercontent.com/42788152/233400828-b81ca080-1adf-4a66-b5ad-fa64e4305bc1.png)

![image](https://user-images.githubusercontent.com/42788152/233400891-a20d183f-e930-476e-8e6c-861002f3ded0.png)

![image](https://user-images.githubusercontent.com/42788152/233400962-7a7f9e46-16d4-4393-9bef-2fe99a1bb4d3.png)

**Comment by Mezatsong** [comment]
Fixed with this PR #8

**Comment by j-tepe** [comment]
@Mezatsong I'm still experiencing this issue on V1.6.1

```
'lines' => 'required|array|min:1',
'lines.*.code_id' => 'required|integer|exists:codes,id',
'lines.*.user_id' => 'required|integer|exists:users,id',
'lines.*.description' => 'string|max:255',
```
